src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -0,0 +1,72 @@
+package ui
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"path"
+	"path/filepath"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/tarfs"
+	"github.com/skratchdot/open-golang/open"
+)
+
+const (
+	localResourcesPath = "../ui/app"
```

Comment:
@atavism for whatever reason I see the same thing when running with ```go run``` but not with the precompiled ./flashlight. So this works for me:

```
./flashlight -role client -addr localhost:10080 -uiaddr localhost:16785 -cloudconfig http://s3.amazonaws.com/lantern_config/cloud.valencia.yaml.gz
```